facebookads/objects.py

```diff
@@ -1357,15 +1358,24 @@ def get_endpoint(cls):
         return 'adimages'
 
     @classmethod
-    def remote_create_from_zip(cls, filename, parent_id, api=None):
+    def remote_create_from_zip(cls, file=None, parent_id=None, api=None,
+                               filename=None):
+        if filename:
+            warnings.warn("The 'filename' argument has been depricated in "
+                          "favor of 'file'", DeprecationWarning, stacklevel=2)
+        file = file or filename
```

Comment:
This may cause unexpected behavior for someone who is not diligent with understanding the different parameters. Instead, let's throw an error if the both `file` or `filename` are provided. In the event someone does it (where they represent different files), we silently select one for them where we should probably just complain.

```
if filename:
    # warn
    file = filename
    if file:
        throw

if isinstance(file, basestring):
    ...
```